site/public/js/server.js

```diff
@@ -1096,6 +1096,12 @@ function enableTabsInTextArea(jQuerySelector) {
     });
 }
 
+function confirmBypass(str, redirect){
```

Comment:
This file is not linted by `eslint`, but the linter would complain about this.

```suggestion
function confirmBypass(str, redirect) {
```